certified-connectors/iObeya/apiProperties.json

```diff
@@ -0,0 +1,106 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "{ClientId}",
+          "scopes": [
+            "api offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrlTemplate": {
+              "value": "https://{HostnamePrefix}.iobeya.com/s/oauth/authorize"
+            },
+            "tokenUrlTemplate": {
+              "value": "https://{HostnamePrefix}.iobeya.com/s/oauth/token"
+            },
+            "refreshUrlTemplate": {
+              "value": "https://{HostnamePrefix}.iobeya.com/s/oauth/token"
+            }
+          }
+        }
+      },
+      "token:HostnamePrefix": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 0,
+            "required": "true"
+          },
+          "description": "Prefix of your iObeya Server, ex : 'demo' for demo.iobeya.com",
+          "displayName": "iObeya Server",
+          "tooltip": "Prefix of your iObeya Server, ex : 'demo' for demo.iobeya.com"
+        }
+      },
+      "token:ClientId": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 1,
+            "required": "true"
+          },
+          "description": "Your API Client ID",
+          "displayName": "Client ID",
+          "tooltip": "The API Client ID created in iObeya Administration > Settings > API"
+        }
+      },
+      "token:ClientSecret": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 2,
+            "required": "true"
+          },
+          "description": "Your API Client Secret",
+          "displayName": "Client Secret",
+          "tooltip": "The API Client Secret created in iObeya Administration > Settings > API"
+        }
+      }
+    },
+    "iconBrandColor": "#f8faff",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setproperty",
+        "title": "Create a nextLink",
```

Comment:
@smiquerolle strings with camel case are considered non-translatable. If you want to localize the string, the camel case should be fixed here. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).